WordPress/pattern-directory
# Try `@wordpress/edit-post` for the creator UI.

This PR is an experiment to test the use of `import { initializeEditor } from '@wordpress/edit-post';`

@ryelle Since I didn't author all the changes in the branch, can you take a really critical eye? 

Closes #59.

**Implements**:
- Various style updates

**Doesn't yet Implement**:
- Tie into Publish window #112

### Screenshots
![](https://d.pr/i/Gpvb48.png)

### How to test the changes in this Pull Request:
- Visit `/new-pattern/`
- Add a title
- Add some blocks
- Add a description
- Select some categories
- Change the viewport width
- Select block scope

In the browser console, paste the following one by one:

```
wp.data.select('core/editor').getEditedPostAttribute('title')
wp.data.select('core/editor').getEditedPostAttribute('pattern-categories')
wp.data.select('core/editor').getEditedPostAttribute('meta')
```
Expect to see your updates values.
